trufflehog.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: trufflehog
+  version: 3.88.7
+  epoch: 0
+  description: Find, verify, and analyze leaked credentials
+  dependencies:
+    runtime:
+      - bash
+      - binutils
+      - ca-certificates
+      - cpio
+      - git
+      - openssh-client
+      - rpm2cpio
+  copyright:
+    - license: AGPL-3.0-only
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/trufflesecurity/trufflehog
+      tag: v${{package.version}}
+      expected-commit: f2dc96e8548d5fab0f6ffb3d47b9918b49fdde0a
+
+  - uses: go/build
+    with:
+      packages: ./main.go
+      ldflags: -X github.com/trufflesecurity/trufflehog/v3/pkg/version.BuildVersion=${{package.version}}
+      output: trufflehog
+
+  - runs: |
+      mkdir -p ${{targets.contextdir}}/etc
+      cp entrypoint.sh ${{targets.contextdir}}/etc/entrypoint.sh
```

Comment:
Can we use `install -Dm 755` here and omit the `chmod` below?

Reply:
thanks for the catch here!! +1 on using more of install here.